### PR TITLE
[ENDPOINT]: POST /members - OT102-90

### PR DESCRIPTION
**Pull request by JonatanGarbuyo**
[OT102-90](https://alkemy-labs.atlassian.net/browse/OT102-90)
## Description:

AS user
I WANT to create a member
TO update the information on the web.

## Acceptance criteria:
POST / members - You must validate that the name field is sent, and it is a string

### CODE REVIEW

File: services/members.js [hunk 1]
```diff
@@ -7,3 +8,23 @@ exports.getMembers = async () => {
...
+exports.addMember = async (data) => {
+  try {
+    const { id, name, image } = await Member.create(data)
+    return { id, name, image }
+  } catch (error) {
+    if (error.name === 'SequelizeUniqueConstraintError') {
+      throw new ErrorObject(
+        '[Error creating member] [] - The member already exist',
+        409, // status 409 Conflict
+        error.errors,
```
Comment (dvelezroman) [inline services/members.js]: `const httpError = createHttpError(500, `[Error retrieving users] - [users - get]: ${error.message}`)
      httpError.errors = errors
      next(httpError)`

File: services/members.js [hunk 2]
```diff
@@ -7,3 +8,23 @@ exports.getMembers = async () => {
...
+      )
+    }
+    throw new ErrorObject(
```
Comment (dvelezroman) [inline services/members.js]: `const httpError = createHttpError(500, [Error retrieving users] - [users - get]: ${error.message}) httpError.errors = errors next(httpError)`

**Review by dvelezroman — CHANGES_REQUESTED** [review]

**Review by dvelezroman — APPROVED** [review]
Buen PR

**Review by gabyMangoni — APPROVED** [review]
Approve

**Review by gabodmc — APPROVED** [review]

**Pull request merged**